test/utils/utils.go

```diff
@@ -473,7 +475,9 @@ func Containerized() bool {
 }
 
 func init() {
-	rand.Seed(GinkgoRandomSeed())
+	seed := GinkgoRandomSeed()
+	logrus.Warnf("rand.Seed @ test/utils/utils.go:init %#v", seed)
+	rand.Seed(seed)
```

Comment:
I think we should use `seed := time.Now().UnixNano()` here.  There is value in ginkgo randomizing test order and providing a `-seed` option to control that.  However, I don't think it's useful to have "predictable" random number clashes within the tests.  In other words, I see more danger than value in having (for example) a deterministic random port.

IMHO, we can probably drop the logging of the seed here as well, I think it's just extra noise at this point.